validator/delphi_validator/static.py

```diff
@@ -84,10 +83,11 @@ def check_missing_date_files(self, daily_filenames, report):
         Returns:
             - None
         """
-        number_of_dates = self.end_date - self.start_date + timedelta(days=1)
+        number_of_dates = self.time_window.end_date - self.time_window.start_date +\
```

Comment:
Should this and the `date_seq` belong as attributes in the TimeWindow object?

Reply:
Changed this to derive from the `TimeWindow.span_length` member.  I can do more if you'd like.

Reply:
I think adding the date_seq to TimeWindow  too would make sense (bit of a nit), since then the TimeWindow tells you the start, end, dates included, length, etc.

Reply:
Yes, done